pkgs/games/gdlauncher/default.nix

```diff
@@ -0,0 +1,30 @@
+{ pkgs, lib, ... }:
+let
+  pname = "gdlauncher";
+  version = "1.1.30";
+  name = "${pname}-${version}";
+  src = pkgs.fetchurl {
+    url = "https://github.com/gorilla-devs/GDLauncher/releases/download/v${version}/GDLauncher-linux-setup.AppImage";
+    hash = "sha256-4cXT3exhoMAK6gW3Cpx1L7cm9Xm0FK912gGcRyLYPwM=";
+  };
```

Comment:
please use the `fetchFromGitHub` wrapper function. https://nixos.org/manual/nixpkgs/stable/#fetchfromgithub

Reply:
I need fetchurl to get the appimage from the release.